### PR TITLE
Avoid duplicated citation warnings

### PR DESCRIPTION
**Pull request by acse-ej321**
- Re-implements fix for #7 , overriding margins, as a separate static CSS file to overwrite the sphinx theme.
- Address #9 by removing the duplicate reference section in API page source documents and replacing with a hyperlink to a stand-alone, common reference section.
- Addressing #9 also addressed #11

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -62,43 +84,23 @@ html: apidoc $(GENERATED_FILES)
 	cp -r $(BUILDDIR)/html/* ..
 
 .PHONY: apidoc
-apidoc: apidoc-animate apidoc-movement apidoc-goalie
+
+apidoc: 
+
+# apidoc-animate apidoc-movement apidoc-goalie
```
Comment (jwallwork23) [inline docs/Makefile:90]: Do we need this comment?

File: docs/source/conf.py [hunk 2]
```diff
@@ -107,7 +111,15 @@
...
-html_static_path = []
+html_static_path = ['_static/custom.css']
+
+# Add custom static files (such as style sheets) to override all style themes. 
+# This is sphinx specific and not a theme-specific option. 
+# reference:
+# https://stackoverflow.com/questions/23462494/how-to-add-custom-css-file-to-sphinx
+html_context = {
+    'css_files': ['_static/custom.css'],
+    }
```
Comment (jwallwork23) [inline docs/source/conf.py:122]: Please could you apply Black to these files to auto-format?

File: docs/source/references.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+References
+----------
+.. rubric:: References
+
+.. bibliography:: references.bib
+    :all:
+
+
+
```
Comment (jwallwork23) [inline docs/source/references.rst:9]: Do we need these trailing blank lines?

**Review by jwallwork23 — CHANGES_REQUESTED** [review]
Thanks very much for these fixes! Just some minor requests.

File: docs/Makefile (hunk 1 above)
Comment (acse-ej321) [reply docs/Makefile]: this was an unintentional edit, changed back to the original Makefile format

File: docs/source/references.rst (hunk 3 above)
Comment (acse-ej321) [reply docs/source/references.rst]: trailing lines removed

**Review by jwallwork23 — APPROVED** [review]
Looks good! If you could do `make html`, commit the changes, push and then merge that'd be great.

**Pull request merged**